pkg/operator/http.go

```diff
@@ -90,19 +88,17 @@ func newRouter(
 		reportResultsGetter:          reportResultsGetter,
 		namespace:                    namespace,
 		reportLister:                 reportLister,
-		scheduledReportLister:        scheduledReportLister,
 		reportGenerationQuerieLister: reportGenerationQuerieLister,
 		prestoTableLister:            prestoTableLister,
 	}
 
-	router.HandleFunc(APIV1ReportsGetEndpoint, srv.getReportHandler)
-	router.HandleFunc("/api/v2/reports/{name}/full", srv.getReportV2FullHandler)
-	router.HandleFunc("/api/v2/reports/{name}/table", srv.getReportV2TableHandler)
+	router.HandleFunc(APIV2ReportsEndpointPrefix+"/{name}/full", srv.getReportV2FullHandler)
```

Comment:
do we have API docs for these?

Reply:
Uh sorta, not really. There's some examples of this output in the api.md docs.